### ISSUE TITLE
www-plugins/opera-ffmpeg-codecs broken chromium-FORTIFY_SOURCE-r2.patch

### ISSUE THREAD

**Issue by tslnox** [jorgicio/jorgicio-gentoo#161]

**Comment by tslnox** [comment]
Sorry for making such a mess, I am not accustomed to the issues tracker there yet.

I have found that opera-ffmpeg-codecs-71.0.3578.98 doesn't build due to the chromium-FORTIFY_SOURCE-r2.patch pointing to different lines (I guess that even though the patched portion didn't change, it changed position). I have fixed the lines and now the build is going on. I will update as soon as it builds or not. The fixed file is there: 
[chromium-FORTIFY_SOURCE-r2.patch.txt](https://github.com/jorgicio/jorgicio-gentoo/files/2998482/chromium-FORTIFY_SOURCE-r2.patch.txt)
